dags/iop/parser.py

```diff
@@ -183,3 +188,29 @@ def _get_country(self, article, id):
         except AttributeError:
             self.logger.error("Country is not found in XML")
             return
+
+    def _extract_copyright_year(self, article):
+        try:
```

Comment:
also maybe don't use attributeError and check if `article.find` is `None`

Reply:
I like the solution for catching the AttributeError more. Before, I was using if/esle clauses, however,  `article.find(path) is nor None` were really confusing to everyone who checked the PR. Attribute error looks easier to understand, don't you think?